uni/lib/controller/parsers/parser_schedule_html.dart

```diff
@@ -64,6 +70,45 @@ Future<List<Lecture>> getScheduleFromHtml(http.Response response) async {
       semana = semana.expand((i) => [(i - 1) < 0 ? 0 : i - 1]).toList();
     }
   });
+  final overlappingClasses = document.querySelectorAll('.dados > tbody > .d');
+  for (final element in overlappingClasses) {
+    final String? subject = element.querySelector('acronym > a')?.text;
+    final String? typeClass = element.querySelector('td[headers=t1]')?.nodes[2].text
+        ?.trim().replaceAll(RegExp(r'[()]+'), '');
+    final String? textDay = element.querySelector('td[headers=t2]')?.text;
+    final int? aDay = document.querySelector('.horario > tbody > tr:first-child')
+        ?.children
+        .indexWhere((element) => element.text == textDay);
+    final int day = aDay != null ? aDay - 1 : 0;
+    final startTime = element.querySelector('td[headers=t3]')?.text;
+    final room = element.querySelector('td[headers=t4] > a')?.text;
+    final teacher = element.querySelector('td[headers=t5] > a')?.text;
+    final classNumber = element.querySelector('td[headers=t6] > a')?.text;
```

Comment:
Declare this variables as String, the end result is the same, but it's more readable. Not the problem here, but in some cases some time is lost figuring out the type of the variable